docs/src/components/SubscriptionForm.astro

```diff
@@ -85,6 +85,13 @@ const { data } = await getEntry('i18n', lang ?? 'en');
       text-decoration: none;
       font-size: var(--sl-text-sm);
       background: var(--sl-color-text-accent);
+      transition: var(--button-transition);
```

Comment:
Added `transition` here to also show the out-animation, without these, only the hover is animated but not on unhovering.